lib/src/models/chat/agent_feedback_info.dart

```diff
@@ -78,17 +78,23 @@ class AgentFeedbackInfo {
 }
 
 /// The state of the feedback.
+/// The states can be found in the backend under https://github.com/aira/aira-backend-services/blob/main/api-models/src/main/resources/openapi/task-manager.yaml#L693-L697
 enum AgentFeedbackState {
   pending('PENDING'),
-  inProgress('IN_PROGRESS'),
+  assigned('ASSIGNED'),
+  started('STARTED'),
   completed('COMPLETED');
 
   final String value;
 
   const AgentFeedbackState(this.value);
 
   /// Gets the [AgentFeedbackState] from the given value.
-  static AgentFeedbackState? fromValue(String? value) {
-    return AgentFeedbackState.values.firstWhereOrNull((e) => e.value == value);
+  ///
+  /// If the value is not found, [pending] is returned.
+  static AgentFeedbackState fromValue(String? value) {
+    return AgentFeedbackState.values
+            .firstWhereOrNull((e) => e.value == value) ??
```

Comment:
why not `AgentFeedbackState.value.firstWhere((e) => e.value == value, orElse: () => pending)` ???

Reply:
I agree, it doesn't look as cleaner now as it used when i was returning null for not found. I will change it just to make you satisfied :D

Reply:
I agree, it doesn't look as cleaner now as it used when i was returning null for not found. I will change it just to make you satisfied :D